recipes/playrho/all/conanfile.py

```diff
@@ -0,0 +1,103 @@
+
+    def export_sources(self):
+        self.copy("CMakeLists.txt")
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            self.copy(patch["patch_file"])
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    @property
+    def _compilers_minimum_versions(self):
+        return {
+            "gcc": "8",
+            "Visual Studio": "16",
+            "clang": "7",
+            "apple-clang": "12",
+        }
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 17)
+
+        compilers_minimum_version = self._compilers_minimum_versions
+        minimum_version = compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version:
+            if tools.Version(self.settings.compiler.version) < minimum_version:
+                raise ConanInvalidConfiguration("{} requires C++17, which your compiler does not support.".format(self.name))
+        else:
+            self.output.warn("{} requires C++17. Your compiler is unknown. Assuming it supports C++17.".format(self.name))
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+            destination=self._source_subfolder, strip_root=True)
+
+    @functools.lru_cache(1)
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["PLAYRHO_BUILD_SHARED"] = self.options.shared
+        cmake.definitions["PLAYRHO_BUILD_STATIC"] = not self.options.shared
+        cmake.definitions["PLAYRHO_INSTALL"] = True
+        cmake.configure()
+        return cmake
+
+    def build(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy(pattern="LICENSE.txt", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "PlayRho"))
+
+    def package_info(self):
+        self.cpp_info.libs = ["PlayRho"]
+
+        if self.settings.os in ["Linux", "FreeBSD"]:
+            self.cpp_info.system_libs.append("m")
+
+        self.cpp_info.set_property("cmake_file_name", "PlayRho")
+        self.cpp_info.set_property("cmake_target_name", "PlayRho::PlayRho")
+
+        #  TODO: to remove in conan v2 once cmake_find_package_* generators removed
+        self.cpp_info.filenames["cmake_find_package"] = "PlayRho"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "PlayRho"
+        self.cpp_info.names["cmake_find_package"] = "PlayRho"
+        self.cpp_info.names["cmake_find_package_multi"] = "PlayRho"
```

Comment:
Imported target name is wrong: https://github.com/louis-langholtz/PlayRho/blob/v1.1.0/PlayRho/CMakeLists.txt#L146-L157

- not namespaced
- static: `PlayRho`
- shared: `PlayRho_shared`

Reply:
A [branch and pull request for these changes against the 1.1.0 release of PlayRho](https://github.com/louis-langholtz/PlayRho/pull/435) is now available as well. I plan to make that into a 1.1.1 release if I pull it. The changes particular to the issue @SpaceIm opened are in commit [96ffa823](https://github.com/louis-langholtz/PlayRho/pull/435/commits/96ffa823360917e1eeac72e965e3cf8543dbacf0).

If someone here could specifically give this commit (or branch or PR) a try, that'd be awesome!